src/components/PageLoading/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Spinner from 'components/Spinner';
+
+const PageLoading = () => {
+  const { t } = useTranslation('general');
+  return (
+    <div className="margin-y-10" data-testid="page-loading">
+      <div className="text-center">
+        <Spinner size="xl" aria-valuetext={t('pageLoading')} aria-busy />
```

Comment:
Wondering whether you'd ever want to be able to adapt the size of the PageLoader and to have some size options here?